### PR TITLE
fix issue #100 fixed row labels names for `hypothesis`= `revpairwise`, `pairwise`, `revreference`, `reference`, `revsequential`, `sequential` 

### PR DESCRIPTION
**Pull request by artiom-matvei**
Fix for issue #100 

Issue mentions `revpairwise` and `pairwise`, but this fixes `revreference`, `reference`, `revsequential` and `sequential` as well.

### CODE REVIEW

**Comment by artiom-matvei** [comment]
@vincentarelbundock please, have a look and let me know if you have any comments.

File: tests/test_jss.py [hunk 1]
```diff
@@ -6,114 +6,175 @@
...
+            variables={"equal": [30, 90]},
+            hypothesis=h,
+        )
+        assert isinstance(c, pl.DataFrame)
+        if b == "democracy":
+            assert c.shape[0] == 1
+            assert c["term"][0] == label[b]
+        else:
+            assert c.shape[0] > 1
+            assert (c["term"] == label[b]).all()
```
Comment (artiom-matvei) [inline tests/test_jss.py:172]: @vincentarelbundock do you agree on testing the labels here?

Comment (vincentarelbundock) [reply tests/test_jss.py:172]: yes, that's fine.

**Comment by vincentarelbundock** [comment]
thanks a lot for this. merged

**Pull request merged**